bindings/java/src/main/java/org/ucx/jucx/ucp/UcpWorkerParams.java

```diff
@@ -39,13 +39,13 @@ public UcpWorkerParams clear() {
      * should be created with. The default value is UCS_THREAD_MODE_SINGLE and
      * it is used when the value of the parameter is not set. When this
      * parameter is set, the {@link UcpWorker#UcpWorker(UcpContext)}
-     * attempts to create worker with this thread mode.
+     * attempts to create worker where multiple threads can access concurrently.
      * The thread mode with which worker is created can differ from the
      * suggested mode.
      */
-    public UcpWorkerParams setThreadMode(int threadMode) {
+    public UcpWorkerParams setMultiThreadMode() {
```

Comment:
from this perspective, better rename to `requestThreadSafety` for consistency...